operate/client/src/modules/queries/variables/useVariables.ts

```diff
@@ -16,12 +16,16 @@ import {useDisplayStatus} from 'modules/hooks/variables';
 const MAX_VARIABLES_PER_REQUEST = 50;
 const VARIABLES_SEARCH_QUERY_KEY = 'variablesSearch';
 
+function getQueryKey(processInstanceKey: string, scopeId: string | null) {
+  return [VARIABLES_SEARCH_QUERY_KEY, processInstanceKey, scopeId];
+}
```

Comment:
Better to give a more descriptive name or use the approach I mentioned through Slack